### PR TITLE
Add silo for Yearn vaults

### PR DESCRIPTION
**Pull request by haydenshively**

### CODE REVIEW

File: tests/yUSDC_USDC_WETH_nop.test.js [hunk 1]
```diff
@@ -0,0 +1,330 @@
+const chai = require("chai");
```
Comment (adamegyed) [inline tests/yUSDC_USDC_WETH_nop.test.js:1]: File name should be "yvUSDC"

File: tests/yUSDC_USDC_WETH_nop.test.js [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+  });
+
+  it("should withdraw rest", async () => {
+    let shares = (await aloeBlend.balanceOf(WHALE)).toNumber();
+    await aloeBlend.withdraw(shares.toString(), 0, 0, { from: WHALE });
+
+    console.log((await token0.balanceOf(aloeBlend.address)).toString(10));
+    console.log((await token1.balanceOf(aloeBlend.address)).toString(10));
+    console.log((await silo0.balanceOf(aloeBlend.address)).toString(10));
+    console.log((await silo1.balanceOf(aloeBlend.address)).toString(10));
```
Comment (adamegyed) [inline tests/yUSDC_USDC_WETH_nop.test.js:328]: These should be expect statements

**Review by adamegyed — COMMENTED** [review]
Few smol updates

File: contracts/silos/YearnSilo.sol [hunk 3]
```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: Unlicense
```
[the remark this answers is not in the document]
Comment (haydenshively) [reply contracts/silos/YearnSilo.sol]: Done

File: tests/yUSDC_USDC_WETH_nop.test.js (hunk 1 above)
Comment (haydenshively) [reply tests/yUSDC_USDC_WETH_nop.test.js]: Done

**Pull request merged**